### ISSUE TITLE
Fix font display on small screens

### ISSUE THREAD

**Issue by Popolechien** [kiwix/kiwix-tools#286]
See #285. There is a display/readibility issue on smaller screen (e.g. KaiOS/Nokia Banana phone) that makes accessing content rather difficult. It would be nice to use smaller fonts when on a small screen so as to make the content usable (Google has already done the optimization work, so that could be a reference to use).

**Comment by kelson42** [comment]
The visible font size of a device screen depends on:
* The size specified in the HTML
* The level of zoom of the browser (in the Kiwix reader)
* The pixel-depth of the screen

Here I have no clue  where the problem is. We need to give an answer to all 3 questions before starting anything else. One time we have this we also need to know what would be a good size (or good sizes).

**Comment by kelson42** [comment]
@ISNIT0 Please tell me here what is the strategy to solve this problem. To me it looks like quite complicated. We need to an overall strategy to scale all texts based on the pixel-density + resolution.

**Comment by kelson42** [comment]
We believe fixing #825 was the only thing to do there.
